go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/maticnetwork/heimdall
 
-go 1.20
```

Comment:
Any reason why we are downgrading to `go 1.19 `?

Reply:
I had just updated it to perform go mod tidy as locally I had go 1.19, have reverted it back